AdminWebsite/AdminWebsite/ClientApp/src/app/services/video-hearings.service.ts

```diff
@@ -441,4 +445,18 @@ export class VideoHearingsService {
     getJudge(): ParticipantModel {
         return this.modelHearing.participants.find(x => x.is_judge);
     }
+
+    isConferenceClosed(): boolean {
+        return this.modelHearing.status === BookingStatus.Created && this.modelHearing.telephone_conference_id === '';
+    }
+
+    isHearingAboutToStart(): boolean {
+        if(this.modelHearing.scheduled_date_time && this.modelHearing.status) {
```

Comment:
this.modelHearing.status will only be populated if it is not a new hearing(so when editing). Based on Lorenzo's comments elsewhere this should be changed to only be allowed if the booking is confirmed which is not currently available on HearingModel.